package.json

```diff
@@ -85,6 +85,7 @@
     "sharp": "^0.32.0",
     "source-map-loader": "^4.0.1",
     "style-loader": "3.3.3",
+    "ts-jest": "^26.5.0",
     "typescript": "^4.9.4",
```

Comment:
consider: Let's ensure all these `dependencies` are pinned to explicit version like the others.

Reply:
fair